### PR TITLE
Direct upload integration tests

### PR DESCRIPTION
**Pull request by mbarton**
Ronseal.

- Create an atom
- Upload a YouTube video to it
- Delete both afterwards

Also adds a flag to initial upload creation to stop the video getting sent to Pluto.

### CODE REVIEW

File: common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala [hunk 1]
```diff
@@ -102,21 +102,24 @@ abstract class UploadActionHandler(store: UploadsDataStore, plutoStore: PlutoDat
...
     val plutoData: PlutoSyncMetadata = upload.metadata.pluto
 
-    plutoData.projectId match {
-      case _ if plutoData.assetVersion == -1 =>
+    if(plutoData.enabled) {
+      log.info(s"Not syncing to Pluto upload=${upload.id} atom=${plutoData.atomId}")
+    } else {
+      plutoData.projectId match {
+        case _ if plutoData.assetVersion == -1 =>
         // TODO: work out what to do here? probably need to manually add the asset
```
Comment (jharewinton) [inline common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala]: Does this need to be finished here?

File: integration-tests/src/test/scala/VideoUploadTests.scala [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    atomId = (Json.parse(response.body().string()) \ "id").get.as[String]
+
+    addAtomToStore(atomId)
+
+    apiEndpoint = apiUri(atomId)
+
+    eventually {
+      gutoolsGet(apiEndpoint).code() should be(200)
+    }
```
Comment (jharewinton) [inline integration-tests/src/test/scala/VideoUploadTests.scala]: This completely duplicates the steps of the atom creation tests. Might be good to pull this out to a separate function, probably in IntegrationTestBase, for easier maintenance

File: common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala (hunk 1 above)
Comment (mbarton) [inline common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala]: I'm not sure we even need this field in the pluto metadata at all anymore

Comment (mbarton) [inline common/src/main/scala/com/gu/media/upload/actions/UploadActionHandler.scala]: We do still need it but the pluto stuff doesn't need to care. I've removed that case

File: integration-tests/src/test/scala/VideoUploadTests.scala (hunk 2 above)
Comment (mbarton) [inline integration-tests/src/test/scala/VideoUploadTests.scala]: I've had a go at refactoring it. Let me know what you think @jharewinton

Comment (jharewinton) [inline integration-tests/src/test/scala/VideoUploadTests.scala]: 👍

**Pull request merged**